README.md

```diff
@@ -239,6 +239,12 @@ DotNet:
 instrumentation.opentelemetry.io/inject-dotnet: "true"
 ```
 
+Apache HTTPD:
+```bash
+instrumentation.opentelemetry.io/inject-apache-httpd: "true"
+instrumentation.opentelemetry.io/otel-namespace: "my-custom-namespace" # optional, if missing, OTEL_NAMESPACE will be set to pod namespace 
```

Comment:
Yes, by default, it is set to httpd pod namespace. 
Some backends may use OTEL_NAMESPACE value in a different context than K8S namespace, like AppDynamics use it for "application name" and users may want to set it to anything they consider application name.

Reply:
Is the value of the annotation exclusively used to set `OTEL_NAMESPACE` ? If so could users use the resource section in the CR?

Reply:
It's more importantly used in a configuration file of httpd which get's generated as part of the instrumentation - it's one of the mandatory attributes for the webserver instrumentation library. The value from a CR would apply to all deployments of httpd using that Instrumentation CR, in practice requiring potentially a lot of those, each "application specific". 

I found it more logical to have it specified as part of the actual deployment/pod spec and I needed some mechanism to do so. 

What do you think?

Reply:
Could we move this to a separate issue/PR? 

I am concerned of adding this annotation and especially just for the HTTPD.

Reply:
I totally get that. 

And thinking of it - it is actually generic topic - if someone would, say, have a Java application and want to trace it via AppDynamics, where OTEL_NAMESPACE is required, it would have to be set either on CR level or in the application manifest. 

So I'll remove it for the time being and let it on the K8S namespace as a default. And let's think of a better way of doing it generically.

Reply:
I'll open an issue for this.

Reply:
annotation removed, tested, issue for discussion raised at https://github.com/open-telemetry/opentelemetry-operator/issues/1572

Reply:
@pavolloffay - in your opinion - is it ok like this, i.e. for the time being, using K8S namespace as default for OTEL_SERVICE_NAMESPACE? If so, can we move this forward?

Reply:
@pavolloffay - can we proceed with this PR?